hadoop-hdfs-project/hadoop-hdfs-rbf/src/test/java/org/apache/hadoop/hdfs/server/federation/router/TestRouterAdminCLI.java

```diff
@@ -700,6 +707,46 @@ public void testAddMountTableIfParentExist() throws Exception {
     }
   }
 
+  @Test
+  public void testInitViewFsToMountTable() throws Exception {
+    // re-set system out for testing
+    System.setOut(new PrintStream(out));
+    stateStore.loadCache(MountTableStoreImpl.class, true);
+    String nnAddress = cluster.getRandomNamenode().getNamenode().getHostAndPort();
+
+    String src = "/data";
+    Path destPath = new Path("hdfs://" + nnAddress + "/data");
+    String user = "user1";
+    String group = "group1";
+    String clusterName = "ClusterX";
+
+    // 0.mkdir destPath
+    hdfs.mkdirs(destPath);
+    // 1.set owner
+    hdfs.setOwner(destPath, user, group);
+    // 2.set viewFs mapping
+    admin.getConf().set("fs.viewfs.mounttable.ClusterX.link." + src, destPath.toString());
```

Comment:
Can we add some documentation to HDFSRouterFederation.md?
With pointers to viewfs, etc.